subsys/bluetooth/services/Kconfig.ias

```diff
@@ -0,0 +1,32 @@
+# Bluetooth GATT Immediate Alert Service
+
+# Copyright (c) 2022 Codecoup
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BT_IAS
+	bool "Support for GATT Immediate Alert Service"
```

Comment:
Needs a `select EXPERIMENTAL`

Reply:
There should also be " [EXPERIMENTAL]" at the end of the description.

Reply:
Done